### PR TITLE
New wallet claim abandon command

### PR DESCRIPTION
**Pull request by hackrush01**
Returns only tx in the output

### CODE REVIEW

File: lbrynet/wallet/manager.py [hunk 1]
```diff
@@ -264,7 +264,12 @@ def tip_claim(self, amount, claim_id, account):
     @defer.inlineCallbacks
     def abandon_claim(self, claim_id, txid, nout):
         account = self.default_account
```
Comment (eukreign) [inline lbrynet/wallet/manager.py]: Pass in the `account` as an argument. We need to start moving towards having account argument on all daemon commands.

File: lbrynet/wallet/manager.py [hunk 2]
```diff
@@ -264,7 +264,12 @@ def tip_claim(self, amount, claim_id, account):
...
-        claim = yield account.get_claim(claim_id)
+        if claim_id is not None:
+            claim = yield account.get_claim(claim_id=claim_id)
+        elif txid is not None and nout is not None:
+            claim = yield account.get_claim(txid=txid, nout=nout)
+        else:
+            raise Exception('Must specify claim_id, or txid and nout')
```
Comment (eukreign) [inline lbrynet/wallet/manager.py]: No check for claim being null, should raise an error that claim was not found.

Also, you can reduce the if statements to just: `account.get_claim(claim_id=claim_id, txid=txid, nout=nout)`. Would have the same result but much less code.

File: lbrynet/daemon/Daemon.py [hunk 3]
```diff
@@ -2326,39 +2326,44 @@ def jsonrpc_publish(self, name, bid, metadata=None, file_path=None, fee=None, ti
...
     @defer.inlineCallbacks
-    def jsonrpc_claim_abandon(self, claim_id=None, txid=None, nout=None):
+    def jsonrpc_claim_abandon(self, claim_id=None, txid=None, nout=None, account_id=None):
         """
         Abandon a name and reclaim credits from the claim
 
         Usage:
             claim_abandon [<claim_id> | --claim_id=<claim_id>]
                           [<txid> | --txid=<txid>] [<nout> | --nout=<nout>]
+                          [--account_id=<account_id>]
```
Comment (eukreign) [inline lbrynet/daemon/Daemon.py]: I would like to standardize on using `--account=` across all commands. I'll update the other commands today. Function argument itself should remain `<account_id>`.

File: lbrynet/daemon/Daemon.py [hunk 4]
```diff
@@ -2326,39 +2326,44 @@ def jsonrpc_publish(self, name, bid, metadata=None, file_path=None, fee=None, ti
...
 
         Options:
             --claim_id=<claim_id> : (str) claim_id of the claim to abandon
             --txid=<txid> : (str) txid of the claim to abandon
             --nout=<nout> : (int) nout of the claim to abandon
+            --account_id=<account_id> : (str) id of the account to use
```
Comment (eukreign) [inline lbrynet/daemon/Daemon.py]: I would like to standardize on using `--account=` across all commands. I'll update the other commands today. Function argument itself should remain `<account_id>`.

File: lbrynet/daemon/Daemon.py (hunk 3 above)
Comment (hackrush01) [reply lbrynet/daemon/Daemon.py]: Ok, will push that along with the integration test

File: lbrynet/daemon/Daemon.py (hunk 4 above)
Comment (hackrush01) [reply lbrynet/daemon/Daemon.py]: Same as above

**Pull request merged**